services/121-service/src/transaction-job-processors/transaction-job-processors.service.ts

```diff
@@ -134,24 +145,20 @@ export class TransactionJobProcessorsService {
           )?.value as string, // This must be a string. If it is not, the intersolve API will return an error (maybe).
         });
     } catch (error) {
-      await this.createTransaction({
-        amount: input.transactionAmount,
-        registration: registration,
+      await this.processTransactionResult({
```

Comment:
If a transaction fails you also need to update completed last transaction etc

Reply:
Very good point.